plugins/output-elasticsearch/src/main/scala/com/stratio/sparkta/plugin/output/elasticsearch/dao/ElasticSearchDAO.scala

```diff
@@ -50,13 +50,16 @@ trait ElasticSearchDAO extends Closeable {
 
   def defaultIndexMapping: Option[String] = None
 
-  def indexMapping: Option[String] = None
+  def mappingType: Option[String] = None
 
   def getSparkConfig(timeName: String, idProvided: Boolean): Map[String, String] = {
     {
-      if (idProvided) Map("es.mapping.id" -> idField.getOrElse(Output.Id)) else Map("" -> "")
+      if (idProvided)
+        Map("es.mapping.id" -> idField.getOrElse(Output.Id))
+      else
+        Map("" -> "")
     } ++
-      Map("es.nodes" -> nodes, "es.port" -> defaultPort) ++ {
+      Map("es.nodes" -> nodes, "es.port" -> defaultPort, "es.index.auto.create" -> "no") ++ {
       defaultAnalyzerType match {
         case Some(analyzer) => Map("es.index.analysis.analyzer.default.type" -> analyzer)
```

Comment:
Since we create the indices automatically we have to disable index auto creation